index.js

```diff
@@ -27,17 +27,18 @@ var gcpMetadata = {
       }, options)
       delete reqOpts.property
 
-      return request(reqOpts, function (err, res, body) {
-        if (err) {
-          callback(err)
-        } else if (!res) {
-          callback(new Error('Invalid response from metadata service'))
-        } else if (res.statusCode !== 200) {
-          callback(new Error('Unsuccessful response status code'), res)
-        } else {
-          callback(null, res, body)
-        }
-      })
+      return request(reqOpts, {noResponseRetries: 0}, 
+          function (err, res, body) {
+            if (err) {
+              callback(err)
+            } else if (!res) {
+              callback(new Error('Invalid response from metadata service'))
+            } else if (res.statusCode !== 200) {
+              callback(new Error('Unsuccessful response status code'), res)
+            } else {
+              callback(null, res, body)
+            }
+          });
```

Comment:
nit: in the extremely short term (until #9 lands) we are following the standard js style guide which forbids semicolons. Does `npm test` catch this?